CHANGELOG.md

```diff
@@ -21,7 +21,7 @@ We use *breaking :warning:* to mark changes that are not backward compatible (re
 
 - [#7317](https://github.com/thanos-io/thanos/pull/7317) Tracing: allow specifying resource attributes for the OTLP configuration.
 - [#7367](https://github.com/thanos-io/thanos/pull/7367) Store Gateway: log request ID in request logs.
-- [#7361](https://github.com/thanos-io/thanos/pull/7361) Query: pass query stats from remote execution from server to client.
+- [#7361](https://github.com/thanos-io/thanos/pull/7361) Query: pass query stats from remote execution from server to client. Warning: we changed the protobuf of the QueryAPI, if you use `query.mode=distributed` you need to update your client (upper level Queriers) first, before updating leaf Queriers (servers).
```

Comment:
I think the usual thing we do would be to start with `Query: *breaking :warning:* ....`

Reply:
Prepended that to the message. Thx for the review.